go.mod

```
@@ -1,8 +1,6 @@
 module github.com/hashicorp/terraform-equivalence-testing
 
-go 1.23.0
-
-toolchain go1.24.1
+go 1.24.1
```

```suggestion
go 1.24.3
```

I think we should update to the latest version of Go here to include the latest bug and security fixes